tailwind.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  mode: 'jit',
```

Comment:
Uhhh, interesting. Almost got lost for me between all the other stuff. I like 👍

Reply:
Haha true I forgot to tell you but it made sense for some utilities. I like it too!